src/OVAL/probes/oval_fts.c

```diff
@@ -651,6 +662,11 @@ static int process_pattern_match(const char *path, pcre **regex_out)
 #undef TEST_PATH2
 
 OVAL_FTS *oval_fts_open(SEXP_t *path, SEXP_t *filename, SEXP_t *filepath, SEXP_t *behaviors, SEXP_t* result)
+{
+	return oval_fts_open_prefixed(NULL, path, filename, filepath, behaviors, result);
+}
+
+OVAL_FTS *oval_fts_open_prefixed(const char *prefix, SEXP_t *path, SEXP_t *filename, SEXP_t *filepath, SEXP_t *behaviors, SEXP_t* result)
```

Comment:
<3

Reply:
?

Reply:
https://www.internetslang.com/_26lt_3B3-meaning-definition.asp

Reply:
What is wrong?

Reply:
Nothing is wrong, I just commented that I like how you handled avoiding ABI/API break. <3 is a ASCII art for a heart.